Sources/Turf/Geometries/LineString.swift

```diff
@@ -215,4 +215,114 @@ extension LineString {
         
         return closestCoordinate
     }
+
+    private func getSquareDistance(coordinate1: CLLocationCoordinate2D, coordinate2: CLLocationCoordinate2D) -> Double {
```

Comment:
Nit: Only include “get” in the name of an asynchronous method or one with significant side effects. Label the two coordinates `from` and `to`.

Reply:
Fixed.